src/LODVisitor.cpp

```diff
@@ -63,8 +63,8 @@ bool testInFrustum(Frustum const& frustum, BoundingBox<double> const& tb) {
 
     // test if any BB corner is inside the halfspace defined
     // by the current plane
-    for (std::size_t j = 0; j < tbPnts.size(); ++j) {
-      if (glm::dot(normal, tbPnts[j]) >= d) {
+    for (auto& tbPnt : tbPnts) {
```

Comment:
Can we use const& here?

```suggestion
    for (auto const& tbPnt : tbPnts) {
```